datanode/src/main/resources/strategus/runAnalysis.R

```diff
@@ -27,7 +27,8 @@ connectionDetails$connectionString <- function() Sys.getenv("CONNECTION_STRING")
 
 connectionDetailsReference <- Sys.getenv("DATA_SOURCE_NAME")
 
-outputLocation <- '/results'
+outputLocation <- file.path(getwd(), "results")
+dir.create(outputLocation, showWarnings = FALSE)
```

Comment:
I hope we will have enough permissions granted to perform this operation on any host

Reply:
Execution Engine 2.x ensures that for both tarball mode and docker mode.